app/src/main/java/com/example/mysimplecoindeck/ui/CoinsViewModel.kt

```diff
@@ -65,4 +115,16 @@ class CoinsViewModel @Inject constructor(
         data class Error(val exception: Throwable): CoinDetailUiState()
         object Empty : CoinDetailUiState()
     }
+
+    sealed class SearchUiState {
+        data class Success(val coinSuggestions: Response<SearchResponse>): SearchUiState()
```

Comment:
Already added a comment in the Repository/RetrofitApi, no need to pass the Response down, you can directly use `SearchResponse`